projects/igniteui-angular/package.json

```diff
@@ -70,6 +70,7 @@
         "@types/jszip": "^3.1.7",
         "hammerjs": "^2.0.8",
         "jszip": "^3.3.0",
+        "tslib": "^2.0.0",
```

Comment:
The schematics tests error is caused by this addition and that's intended ("Dependency tslib missing in dependencies map") - it should be added to 
https://github.com/IgniteUI/igniteui-angular/blob/87dee68ac655fbc60fad83fb6e886b0daa5bf116/projects/igniteui-angular/schematics/utils/dependency-handler.ts#L24

Odd enough, we didn't have tslint as a dependency before (was marked as peer in previous versions of the library schematic), but reading into https://github.com/angular/angular-cli/commit/a042290726ce9710346f9827a559584c58ba3c3c it's a good thing we didn't have it, since it shouldn't be.

I guess as far as `ng-add` is concerned, this should be ignored